commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
Please note that the tests are failing as the constants files reside in this [PR](https://github.com/edx/commerce-coordinator/pull/129).

```
from commerce_coordinator.apps.commercetools_frontend.constants import COMMERCETOOLS_FRONTEND
E   ModuleNotFoundError: No module named 'commerce_coordinator.apps.commercetools_frontend'
```

I temporarily created these constants files and ran the tests to ensure they were working on my local and all of them passed, so the changes in the above PR are added to mine there should be no issues :slightly_smiling_face: